src/Modules/DisplayTodo/constant.ts

```diff
@@ -0,0 +1,2 @@
+//Todo API
+export const TODO_URL="http://localhost:8000/todo";
```

Comment:
create a separate .env file and learn how we can access those variables.
use it here

Reply:
done